libs/langchain/langchain/embeddings/fastembed.py

```diff
@@ -0,0 +1,101 @@
+from typing import Any, Dict, List, Literal, Optional
+
+import numpy as np
+
+from langchain.pydantic_v1 import BaseModel, Extra, root_validator
+from langchain.schema.embeddings import Embeddings
+
+
+class FastEmbedEmbeddings(BaseModel, Embeddings):
+    """Qdrant FastEmbedding models.
+
```

Comment:
Could we add links to documentations -- it's easier for maintainers to provide support for packages they're not familiar with

```suggestion
    FastEmbed is a lightweight, fast, Python library built for embedding generation.
    See more documentation at: 
    * https://github.com/qdrant/fastembed/
    * https://qdrant.github.io/fastembed/

    To use this class, you must install the `fastembed` Python package.

    `pip install fastembed`
```